tests/src/io.jsonwebtoken/jjwt-orgjson/0.11.5/.gitignore

```diff
@@ -0,0 +1,4 @@
+gradlew.bat
```

Comment:
Is this file needed even though we have the top level `.gitignore`? Same goes for the other 2 `.gitignore` files under `jjwt-jackson` and `jjwt-gson`

Reply:
- This is generated in one step by `./gradlew scaffold --coordinates io.jsonwebtoken:jjwt-jackson:0.11.5`. Need to fix https://github.com/oracle/graalvm-reachability-metadata/blob/master/CONTRIBUTING.md?

Reply:
- In fact, once this file is removed, other people running nativeTest will pass unnecessary files to git. It's not like this PR should solve the problem, I marked it as solved.